identity/README.md

```diff
@@ -218,13 +218,36 @@ message Capability {
     Type type = 1;
   }
 
+  // NetworkFence contains the features of the NetworkFence operation that the
+  // CSI-driver supports.
+  message NetworkFence {
+    // Type describes a CSI Service that CSI-drivers can support.
+    enum Type {
+      // UNKNOWN indicates that the CSI-driver does not support the NetworkFence
+      // operation in the current mode. The CSI-Addons CO plugin will most
+      // likely ignore this node for the NetworkFence operation.
+      UNKNOWN = 0;
+
+      // NETWORK_FENCE indicates that the CSI-driver provides RPCs for a
+      // NetworkFence operation.
+      // The presence of this capability determines whether the CSI-Addons CO
+      // plugin can invoke RPCs that require access to the storage system,
+      // similar to the CSI Controller (provisioner).
```

Comment:
is it really the CSI Controller that will do the fencing, or can it also be a CSI Node?

Reply:
Yes, both can perform network fencing operations due to the reason mentioned [here](https://github.com/csi-addons/spec/pull/31#discussion_r769943052)

Reply:
Both _could_ execute anything, but if that is the appropriate approach is unclear to me. As this is more of a storage backend management operation, I would expect that it is only executed by the CSI Controller (provisioner). There might be good reasons for giving a CSI-NodePlugin the capability to network-fence CIDRs, I am not sure. This can be explained a bit better in the spec.

Reply:
I would like to stick to the controller for now as its a Network fencing. csinode/controller makes more sense of volume level fencing.

Reply:
Added description for the same in the spec. Thanks! @Madhu-1 @nixpanic